aiida/common/log.py

```diff
@@ -155,9 +156,11 @@ def configure_logging(with_orm=False, daemon=False, daemon_log_file=None):
     """
     from logging.config import dictConfig
 
+    from aiida.manage.configuration import get_config_option
+
     # Evaluate the `LOGGING` configuration to resolve the lambdas that will retrieve the correct values based on the
     # currently configured profile. Pass a deep copy of `LOGGING` to ensure that the original remains unaltered.
-    config = evaluate_logging_configuration(copy.deepcopy(LOGGING))
+    config = evaluate_logging_configuration(get_logging_config())
```

Comment:
Should update the comment above since there no longer is a deep copy being made explicitly.

Reply:
done